Electron/src/RID/RID.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { Component } from "react"
+import CSS from "csstype"
+import Cameras from "../Core/components/Cameras"
+import Map from "../RED/components/Map"
+import ThreeDRover from "../Core/components/ThreeDRover"
+
+const row: CSS.Properties = {
+  display: "flex",
+  flexDirection: "row",
+  justifyContent: "space-between",
+}
+const column: CSS.Properties = {
+  display: "flex",
+  flexDirection: "column",
+  flexGrow: 1,
+  marginRight: "5px",
+}
+const submod: CSS.Properties = {
+  height: "calc(100% - 30px)",
+  width: "100%",
+  flexGrow: 1,
+}
+
+interface IProps {
+  style?: CSS.Properties
+  rowcol: string
+  onMerge?: (display: string) => void
+  displayed?: string
+}
+
+interface IState {
+  storedWaypoints: any
+  currentCoords: { lat: number; lon: number }
+  display: any
+  displayed: string
+}
+
+class RoverImageryDisplay extends Component<IProps, IState> {
```

Comment:
This whole file needs comments - the recursive aspect of the component is an interesting first for this project, and may be something we want to refer back to this file for in the future, so it should be thoroughly documented.